js/package.json

```diff
@@ -93,8 +93,8 @@
     "slugify": "^1.6.6",
     "source-map": "^0.7.4",
     "uuid": "^9.0.1",
-    "zod": "^3.22.4",
-    "zod-to-json-schema": "^3.22.5"
+    "zod": "3.22.4",
```

Comment:
I can revert this, I believe.

Reply:
Interesting, why is it needed? Would it disrupt any users?

Reply:
i figured it out. explained in a sibling PR.